source/mainnet/smart-contracts/tutorials/piggy-bank/preparing.rst

```diff
@@ -41,7 +41,7 @@ You are now ready to select an instance on the cloud provider that matches the f
    * - Storage
      - 25 GB
    * - Operating system
-     - Ubuntu 20.04 x64
+     - Ubuntu 20.04 or 22.04 x64
```

Comment:
I'm not sure what `x64` is. Probably `x86-64` ?

Reply:
Wikipedia says "x86-64 (also known as x64, x86_64, AMD64, and Intel 64)" ;) But that has been there and I only made minimal changes. I can replace by x86-64 if you prefer.

Reply:
It's also weird that the 64 bit are only mentioned here and not in the other places. Maybe it's better to remove it (or add it to the other places if it is crucial).

Reply:
I am fine with removing it.

Reply:
Although our distribution does not work on ARM :P But that is true for windows as well.

Reply:
But then the x86 is more relevant than the 64 part. ;) Anyway, it's gone.

Reply:
Well it also won't work on 32-bit machines :P